repctl/pkg/k8s/cluster.go

```diff
@@ -366,7 +366,8 @@ func (c *Cluster) CreatePersistentVolumeClaimsFromPVs(ctx context.Context, names
 				Resources: v1.ResourceRequirements{
 					Requests: pv.Requests,
 				},
-				VolumeName:       pv.Name,
+				VolumeName: pv.Name,
```

Comment:
Minor: Why did we lose spacing on this one?

Reply:
Since a comment was added in the next line.